spec/models/aws_engine_spec.rb

```diff
@@ -97,4 +172,71 @@
       expect(value['base64_policy']).to eq(p)
     end
   end
+
+  describe 'SQS' do
+    before do
+      allow(Aws::SQS::Client).to receive(:new).and_return(Aws::SQS::DummyClient.new)
+    end
+
+    it 'succeeds to send message' do
+      expect(AWSEngine.send_message('hoge')).to eq(nil)
+    end
+
+    it 'succeeds to receive message' do
+      expect(AWSEngine.receive_message(10)).to eq([])
+    end
+
+    it 'succeeds to return correct status of the existence of a message' do
+      expect(AWSEngine.message_exist?(nil)).to eq(false)
+      expect(AWSEngine.message_exist?(Aws::SQS::Types::DummyReceiveMessageResult.new)).to eq(true)
+    end
+
+    it 'succeeds to delete message' do
+      expect(AWSEngine.delete_message(Aws::SQS::Types::DummyMessage.new).class.name).to eq("Seahorse::Client::Response")
+    end
+
+    it 'succeeds to delete messages all at once' do
+      expect(AWSEngine.batch_delete([Aws::SQS::Types::DummyMessage.new]).class.name).to eq("Aws::SQS::Types::DeleteMessageBatchResult")
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)